playbooks/prep-node-disks.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: all
+
+  tasks:
+  - name: install dependencies
+    apt: pkg="gdisk"
+  - name: run scan_disks
+    scan_disks:
+
+  - name: check to see what drives are mounted
+    shell: mount | grep {{ item }} | awk '{print $3}'
+    with_items: ssd_devices | union(hdd_devices)
+    ignore_errors: true
+    register: mount_point
+
+  - name: unmount all mounted devices
+    shell: umount {{ item.stdout }}; sed -i '\#{{ item.stdout }}#d' /etc/fstab
+    ignore_errors: true
+    with_items: mount_point.results
+    when: item.stdout != ''
```

Comment:
Is there any way to be more careful about what to unmount here, maybe targeted to specific drives or types?

Also, we try not to do multiple actions in a single task, so the modification of /etc/fstab should happen in a separate task.